### ISSUE TITLE
NVDA does not read the label association for the "Read only" controls present on the page

### ISSUE THREAD

**Issue by harishreddykankanala** [nvaccess/nvda#8786]
### Actual behavior:
NVDA does not read the label association for the "Read only" controls present on the page i.e. Business Unit, Default Queue lookup fields.

### Expected behavior:
NVDA should read the label association for the "Read only" controls present on the page i.e. Business Unit, Default Queue lookup fields.

#### Have you tried any other versions of NVDA? No
[test.zip](https://github.com/nvaccess/nvda/files/2419061/test.zip)

**Comment by Adriani90** [comment]
I can reproduce this in focus mode with NVDA 2019.3 Beta 1.
cc: @MarcoZehe

**Comment by MarcoZehe** [comment]
Not a Firefox bug, Firefox exposes all the associated or aria-label labeled accessible lists as well as options within, even though they are not all focusable. But looking at the tree with both object navigator as well as Accessibility Inspector shows that all of these have names. NVDA reads the options, but not the disabled named list containers.

**Comment by Adriani90** [comment]
This works as expected in NVDA last alpha-31994,b54c62f5 (2024.3.0.31994)
Tested in Firefox 126, and Chrome / Edge 125. Closing as works for me. if you are still having this issue, please comment with a minimal test case and we can reopen.
